packages/styles/primitives/canonical/src/index.css

```diff
@@ -5,6 +5,7 @@
 @import url("normalize.css");
 
 /* Values and settings */
+@import url("@canonical/tokens/dist/css/primitives-canonical.css");
```

Comment:
The `main` of the tokens package is currently `src/index.css`, which doesn't exist.

Perhaps we can create a `index.css` in the dist folder, which imports all of the tokens files.

Or, if that's not possible due to the possibility of some tokens conflicting, we could create import aliases, such as `@canonical/tokens/canonical`, mapped to the dist file you are importing here. This way, imports would't be dependent on internal folder structure.